app/src/main/java/org/mozilla/fenix/settings/DeleteBrowsingDataFragment.kt

```diff
@@ -168,7 +168,11 @@ class DeleteBrowsingDataFragment : Fragment() {
     private fun updateTabCount() {
         view?.open_tabs_item?.apply {
             val openTabs = requireComponents.core.sessionManager.size
+            var clickable = false
+            if (openTabs != 0)
+                clickable = true
             subtitleView.text = resources.getString(R.string.preferences_delete_browsing_data_tabs_subtitle, openTabs)
+            isClickable = clickable
```

Comment:
This could be inlined rather than having a variable.
```suggestion
            isClickable = openTabs > 0
```